RadixWallet/Core/DesignSystem/Components/PlainListRow.swift

```diff
@@ -49,45 +53,82 @@ public struct PlainListRow<Icon: View>: View {
 		subtitle: String? = nil,
 		accessory: ImageAsset? = AssetResource.chevronRight
 	) where Icon == AssetIcon {
-		self.viewState = ViewState(content, rowCoreViewState: .init(title: title, subtitle: subtitle), accessory: accessory)
+		self.viewState = ViewState(content, rowCoreViewState: .init(title: title, subtitle: subtitle), accessory: accessory, hints: [])
 	}
 
 	public var body: some View {
-		HStack(spacing: .zero) {
-			if let icon = viewState.icon {
-				icon
-					.padding(.trailing, .medium3)
-			}
+		VStack(alignment: .leading) {
+			top
+			hints
+		}
+		.padding(.vertical, viewState.rowCoreViewState.verticalPadding)
+		.padding(.horizontal, .medium3)
+		.frame(minHeight: .settingsRowHeight)
```

Comment:
hmm, is there actually a requirement to have a min height? Isn't it about having the same top and bottom padding?

Reply:
This min height actually is necessary for `PlainListRows` that have `kind: .general`. This is, every other row except those used in Settings views. The `minHeight` was already set, and doesn't actually applies for rows with `kind: .settings` as with the vertical padding they will take more space than the min provided. However, if I remove this line, other rows will look smalled and look bad. 

I do agree that the name of the constant isn't appropiate (it is called `settingsRowHeight` when it actually applies to every row but settings ones 🙂). Maybe we rename it to `plainRowMinHeight`?

Reply:
The plain list row has evolved a bit since I introduced it, it's not so plain anymore. Feel free to change the height stuff, just keep in mind that we never want to set absolute heights since we want Dynamic Type to work. Instead we rely on padding outside the text bit as much as possible. An absolute minimum height is fine though, since it won't interfere with Dynamic Type. 

I don't remember what `settingsRowHeight` was back then but it might have broken this rule, so it's great if you clean it up.